datajoint/condition.py

```diff
@@ -183,6 +190,24 @@ def combine_conditions(negate, conditions):
             return not negate  # and empty AndList is True
         return combine_conditions(negate, conditions=items)
 
+    # restrict by Top
+    if isinstance(condition, Top):
```

Comment:
This may be okay for now, but ultimately we should figure out how to collapse multiple Top restrictions into one.

Reply:
Generally, we want to keep the query language algebraically closed. This means that any operation is applicable to any query.